notice/1003/2015-26607_20200101.xml

```diff
@@ -0,0 +1,32 @@
+<notice xmlns="eregs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schemaLocation="eregs https://raw.githubusercontent.com/cfpb/regulations-schema/master/src/eregs.xsd">
```

Comment:
This line fails RegML schema validation, it should be `xsi:schemaLocation` instead of `schemaLocation`.